mappings/net/minecraft/client/render/entity/EntityRendererElytra.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS none/bvm net/minecraft/client/render/entity/EntityRendererElytra
```

Comment:
Should this go somewhere else since it's a renderer that will render on top of an existing player ...? Same for `EntityRendererEndermanBlock` and others.